build.gradle

```diff
@@ -186,11 +186,17 @@ subprojects {
         runtimeClasspath += configurations.compileOnly
       }
       resources {
-        srcDirs = ['test-resources']
+        srcDirs = ['test-resources', rootProject.findProject(':test-utils').file("resource-include-in-all-projects")]
       }
     }
   }
 
+  task cleanLogs(type: Delete) {
```

Comment:
Why does this need to be a separate task? It can go in clean itself.

Reply:
Since tests in each project generates a logs of its own, it's easier to add it to all projects, than to add `logs` to clean of each project.